### ISSUE TITLE
accuracy in val set is different between training and testing 

### ISSUE THREAD

**Issue by Lzc6996** [facebook/fb.resnet.torch#171]
I train a resnet-50 in my own dataset, and i get top1 7.20 in my val set during training. But when i use the testOnly option to test the same model in the same val set i get top1 7.49 .
i want to know what happen there?

**Comment by Lzc6996** [comment]
@colesbury could you help me ? thank you very much! I really need to know what make the same model have different performance. when training i get `* Best model 	7.2048611111111	1.5873015873016` but when i testOnly it turn to top1 error 7.499, the same model in the same dataset.

**Comment by colesbury** [comment]
Possibly differences in batch normalization's running_mean/var between data parallel replicas and the saved model. You can try recomputing the statistics on your training set.

**Comment by Lzc6996** [comment]
I see. I will try. thanks!

**Comment by zaeemzadeh** [comment]
Which one did you report in the papers?
